homeassistant/components/sensibo/number.py

```diff
@@ -0,0 +1,91 @@
+"""Number platform for Sensibo integration."""
+
+from homeassistant.components.number import NumberEntity, NumberEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity import DeviceInfo, EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import SensiboDataUpdateCoordinator
+
+NUMBER_TYPES: tuple[NumberEntityDescription, ...] = (
+    NumberEntityDescription(
+        key="calibration_temp",
+        name="Temperature calibration",
+        icon="mdi:thermometer",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.CONFIG,
+        min_value=-10,
+        max_value=10,
+        step=1,
+    ),
+    NumberEntityDescription(
+        key="calibration_hum",
+        name="Humidity calibration",
+        icon="mdi:water",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.CONFIG,
+        min_value=-10,
+        max_value=10,
+        step=1,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Sensibo number platform."""
+
+    coordinator: SensiboDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    entities = []
```

Comment:
This could be written as a comprehension

Reply:
How would I do that when I have two for where one is in the other one?
Can you make a comprehension inside another one?

Reply:
https://github.com/home-assistant/core/pull/66898/files#r810533305